code/game/objects/machinery/camera/camera.dm

```diff
@@ -234,7 +233,7 @@
 		else
 			myarea = null
 	else
-		set_light(0)
+		set_light(initial(light_range), initial(light_power))
```

Comment:
this is turning the camera off. Why are you changing this to inital in this case?

Reply:
This is for when the AI turns off their lights. It should be setting to initial so the emissives work as discussed (minimum light requirements)